src/main/java/com/codegym/jrugotom5/controller/AdvertController.java

```diff
@@ -27,4 +24,9 @@ public List<AdvertBasicInfoDTO> getAllAdverts() {
     public List<AdvertFullInfoDTO> getAdverts(@RequestParam LocalDate from, LocalDate to) {
         return advertService.getAdvertsByDateRange(from, to);
     }
+
+    @GetMapping("/advert/{id}")
```

Comment:
It's a nitpick, but i would use next URI:
`GET /api/adverts/{id}` and not `/api/adverts/advert/{id}`
Here is a [good guide](https://restfulapi.net/resource-naming/) (see. 1.1 and 2.1.1):